AABB_tree/doc/AABB_tree/Concepts/AABBTraits.h

```diff
@@ -6,12 +6,12 @@
 The concept `AABBTraits` provides the geometric primitive types and methods for the class `CGAL::AABB_tree<AABBTraits>`.
 
 \cgalHasModelsBegin
-\cgalHasModels{CGAL::AABB_traits<AABBGeomTraits,AABBPrimitive>}
+\cgalHasModels{CGAL::AABB_traits_2<AABBGeomTraits_2,AABBPrimitive>}
+\cgalHasModels{CGAL::AABB_traits_3<AABBGeomTraits_3,AABBPrimitive>}
 \cgalHasModelsEnd
 
-\cgalRefines{SearchGeomTraits_3}
+\cgalRefines{SearchTraits}
```

Comment:
This doesn't work: `SearchTraits` is for d dimension, whereas I think you meant you are refining an abstraction of 2 & 3D.

I don't think you need to refine anything, just make sure all types and functors are in this concept (it seems to be the case already).

Reply:
`Search_traits_2` and `Search_traits_3` are both models of `SearchTraits` so it looks good to me. Please hit "Resolve conversation" if you agree